life_expectancy/load_data.py

```diff
@@ -0,0 +1,25 @@
+"""Load data life_espectancy"""
+
+import pandas as pd
+
+def load_data(path: str, delimiter: str = '\t') -> pd.DataFrame:
```

Comment:
```suggestion
def load_data(path: str, *, delimiter: str = '\t') -> pd.DataFrame:
```

Com isto és forçado a usar keyword args ao chamar a função, ex: `load_data(path, delimiter="\t")`
Obriga a ser explícito :)

Reply:
Também tens um comportamento estranho na tua função. 
Se o path for outro que não `.txt`, delimiter não faz nada e não dá nenhum aviso nem erro.

Reply:
Uma opção talvez lançar um erro quando alguém mete delimiter em outro que não `.txt`

Reply:
Sugestões aplicadas! Obrigado!